fission-cli/library/Fission/CLI/Environment/Path.hs

```diff
@@ -14,6 +15,11 @@ import           Fission.Prelude
 
 import           Fission.CLI.Environment.Class
 
+globalDir :: MonadEnvironment m => m FilePath
+globalDir = do
+  global <- getGlobalPath
+  return global
```

Comment:
Two questions:

1. Why this function at all? Naming? If so, should we rename the typeclass method?
2. This is equivalent to `globalDir = getGlobalPath`

Reply:
Good point. The naming of `getGlobalPath` makes sense.

My initial thought was that we wanted to import all paths from `Fission.CLI.Environment.Path`, but less text to read does seem better here. :+1: 

Will push a change to update this to use `getGlobalPath` directly.

Reply:
Updated this to reexport the typeclass method. This gives us the benefit of importing it off `Fission.CLI.Environment.Path` without adding an extra function.